apps/prairielearn/assets/scripts/instructorAssessmentInstancesClient.ts

```diff
@@ -494,6 +494,8 @@ onDocumentReady(() => {
         <button
           class="btn btn-secondary btn-xs ml-1 time-limit-edit-button"
           id="row${row.assessment_instance_id}PopoverTimeLimit"
+          title="Change time limit"
```

Comment:
A title isn't necessary here since it's already being set on when the popover is initialized here: https://github.com/PrairieLearn/PrairieLearn/blob/c2854dfefd7268f8beba478eeb4cc1e43d2fa064/apps/prairielearn/assets/scripts/instructorAssessmentInstancesClient.ts#L72

In the long term, I think we should prefer using `data-title` to declaratively set popover titles so that we don't actually give the element itself a title (even though BS will automatically remove `title` when a popover is initialized). `title` is generally recommended against, see https://developer.mozilla.org/en-US/docs/Web/HTML/Global_attributes/title#accessibility_concerns. Preferring `data-title` over `title` ensures we don't have to second-guess if `title` should be set or not.

Reply:
Removed in [3b5665b](https://github.com/PrairieLearn/PrairieLearn/pull/10496/commits/3b5665b6bc0be4706a91dfd033bf10fb8b256e2e).